### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by TusharBhandirge**

### CODE REVIEW

File: Button Component/button.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+*{
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+    font: 62.5%;
+}
+
+.button{
+    padding: 0.5rem 0.5rem;
```
Comment (gopalbbharadva) [inline Button Component/button.css]: Improvement: Alternative is { padding: 0.5rem}. it will be same.

**Review by gopalbbharadva — APPROVED** [review]
One Suggestion: Give description about PR like What you did , live link or photo gif of the output.
So it will be beneficial for reviewers to review your PR.

File: Button Component/button.css (hunk 1 above)
Comment (TusharBhandirge) [reply Button Component/button.css]: I will make this changes. Thanks

**Review by mimansha-swarup — APPROVED** [review]
looks good to me

**Pull request merged**